pykwasm/src/pykwasm/kdist/wasm-semantics/wasm.md

```diff
@@ -1478,6 +1478,11 @@ Sort `Signedness` is defined in module `BYTES`.
       requires (EA +Int #numBytes(WIDTH)) >Int (SIZE *Int #pageSize())
 
     rule <instrs> load { ITYPE WIDTH EA   Signed DATA:SparseBytes } => #chop(< ITYPE > #signed(WIDTH, #getRange(DATA, EA, #numBytes(WIDTH)))) ... </instrs>
+        [preserves-definedness]
```

Comment:
Feel free to ignore: It may be worth also marking `#chop` as total, it makes it easier to figure out that this rule does, indeed, preserve definedness.

Reply:
Yes, `#chop` should definitely be marekd total... I'll do this on the next PR on Monday.